nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -95,6 +98,7 @@ in
         ExecStart = concatStringsSep " \\\n " (
           [
             "${cfg.package}/bin/k3s ${cfg.role}"
+            "--kubelet-arg=cgroup-driver=systemd"
```

Comment:
This seems to break if docker was started with cgroupfs driver:

```
virtualisation.docker.extraOptions = "--exec-opt native.cgroupdriver=systemd";
```

seems to make it work again. This was on 20.09 so. I need to retest in unstable.

Reply:
TL;DR this is not supported by k3s also see https://github.com/k3s-io/k3s/issues/797 and https://github.com/k3s-io/k3s/blob/8a9c8675c2a16591e13a655617acdb1e75d3962b/pkg/daemons/agent/agent.go#L65